core/dbt/contracts/graph/nodes.py

```diff
@@ -523,6 +525,39 @@ def empty(self):
         """Seeds are never empty"""
         return False
 
+    def disallow_implicit_dependencies(self):
```

Comment:
This looks like a private method, which should be indicated by prefixing with an underscore:
```
def _disallow_implicit_dependencies():
```